mscore/file.cpp

```diff
@@ -493,8 +493,7 @@ void MuseScore::updateNewWizard()
 
 void MuseScore::newFile()
       {
-      if (newWizard == 0)
-            newWizard = new NewWizard(this);
+      updateNewWizard();
```

Comment:
My reason for suggesting the "else" is that this shouldn't be needed in the case where we are creating the wizard for the first time - if it were, then things never would have worked.  Can't build right now to test for myself, but I would think we're now building template list twice the first time wizard is invoked.

Reply:
newFile() instatiated newwizzard, if need be, updateNewWizzard() does that too, and now only it.

Reply:
Never mind, I see now that the previous two lines were removed; I missed that on first read.  That's why I thought something was happening twice.